app/src/main/res/values/strings.xml

```diff
@@ -738,6 +738,8 @@
     <string name="delete_browsing_data_prompt_allow">Delete</string>
     <!-- Text for the snackbar confirmation that the data was deleted -->
     <string name="preferences_delete_browsing_data_snackbar">Browsing data deleted</string>
+    <!-- Text for the snackbar to show the user that the deletion of browsing data is in progress -->
+    <string name="deleting_browsing_data_in_progress">Deleting browsing data…</string>
```

Comment:
I think we want to use `&#8230` for the ellipsize here like in `onboarding_firefox_account_signing_in`